.github/workflows/ci.yml

```diff
@@ -48,7 +48,6 @@ jobs:
     - name: Build rpms
       run: ./ci/rpm.sh
     - name: Upload artidacts
-      uses: actions/upload-artifact@v2
+      uses: ovirt/upload-rpms-action@v2
       with:
-        name: rpm-${{matrix.distro}}
```

Comment:
Does it work for the fedora builds?

Reply:
It should, we are using `$(rpm --eval %{?dist})`

Reply:
Your build looks good except new warning:

     Unexpected input(s) 'path', valid inputs are ['directory']

https://github.com/mwperina/ovirt-imageio/actions/runs/2077619036